packages/react-i18n/src/i18n.ts

```diff
@@ -469,17 +469,18 @@ export class I18n {
       });
     }
 
+    const timeZone = options ? options.timeZone : undefined;
```

Comment:
I'm not sure whether our style guidance allows [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) in `Shopify/quilt`, and hope that the @Shopify/web-foundations  team will chime in if I need to be corrected... but I'm wondering whether
```
  const timeZone = options?.timeZone;
```
would be appropriate here?

Reply:
I had tried that originally, but it causes Jest to throw an error:
> Jest encountered an unexpected token

Reply:
Can you open an issue for that @mmargel? I think we should solve that. Probably we use an old ts-parser for jest.

Reply:
🎉 https://github.com/Shopify/quilt/pull/1643